browser/resources/settings/email_aliases_page/content/email_aliases_signin_page.tsx

```diff
@@ -67,19 +68,29 @@ const BeforeSendingEmailForm = ({ suggestedAuthEmail, emailAliasesService }:
   </SpacedCol>
 }
 
-const AfterSendingEmailMessage = ({ authEmail, emailAliasesService }:
-  { authEmail: string, emailAliasesService: EmailAliasesServiceInterface }) => {
-  const onClick = () => {
+const AfterSendingEmailMessage = (
+  { authEmail, emailAliasesService, errorMessage }:
+  { authEmail: string, emailAliasesService: EmailAliasesServiceInterface,
+    errorMessage: string | undefined }) => {
+  const onClick = (event: React.MouseEvent<HTMLAnchorElement>) => {
+    // Use preventDefault to avoid navigating to the hash
+    event.preventDefault()
     emailAliasesService.cancelAuthenticationOrLogout()
   }
+  const tryAgainLink = (msg: string) => formatMessage(
```

Comment:
super optional but there's also `formatLocale` in `$web-common/locale` which is slightly simpler (but because this is brave-core, works slightly differently :laughing:)

With `formatLocale` this would be:
```tsx
const tryAgainLink = (msg: string) => formatLocale(msg, {
    $1: content => <a href="#" onClick={onClick}>{content}</a>
})
```

(but you have to make the open/close tag use the same placeholder, instead of an incrementing one like `formatMessage` so :shrug:)